### ISSUE TITLE
Implement Saved Queries Functionality

### ISSUE THREAD

**Issue by b-f-chan** [supercluster-covid-data-portal/ui#7]
Epic to capture all the tasks needed to implement saved queries functionality (similar to KidsFirst)

See spec:
https://wiki.oicr.on.ca/display/CC/COVID+Cloud+Data+Explorer+Requirements

**Comment by justincorrigible** [comment]
Moving to Dev column, to account for the code having already been reviewed. The feature itself **has not been deployed** to the environment. The code is available as a feature branch (https://github.com/supercluster-covid-data-portal/ui/tree/feature/query_storage), and should only be merged after the corresponding API ticket for the Query Storage endpoints.

Will move to the QA column once merged into the Develop branch and deployed.

**Comment by b-f-chan** [comment]
Originally wasn't able to convert to epic due to the Zenhub bug, now longer needed nor useful since all the sub-tasks/tickets already completed.  Also no longer blocked by query storage integration ticket.  Closing.
